release-notes/9.0/supported-os.md

```diff
@@ -0,0 +1,128 @@
+# .NET 9 - Supported OS versions
+
+[.NET 9](README.md) is a [Long Term Support (LTS)](../../release-policies.md) release and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```

Comment:
I thought even number releases were LTS only. Was this recently changed?

Reply:
No. That's a copy/paste error.

```suggestion
[.NET 9](README.md) is a [Standard Term Support (STS)](../../release-policies.md) release and [is supported](../../microsoft-support.md) on multiple operating systems per their lifecycle policy.
```

Reply:
Updated the doc with "STS" in-line with our support policy